aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -63,19 +90,28 @@ module aptos_framework::aggregator_v2 {
     }
 
     /// Returns a value stored in this aggregator.
+    /// Note: This operation prevents parallelism of the transaction that calls it.
```

Comment:
let's maybe expand, it prevents parallelism with other transactions that update the aggregator? (maybe also that it is expensive, as expensive and parallelism conflict both are complementary true)